### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by Rachella**
Sorry, I have a few repeats of commit comments - I mistakenly my branch names and thought the push was unsuccessful (and had to do with the commits).  Can you also point me in right direction to look into generating / appending questions dynamically?

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+/*Body and Landing page */
```
Comment (h8rry) [inline css/custom.css:1]: Some indentations are off

File: index.html [hunk 2]
```diff
@@ -8,26 +8,102 @@
...
+            <h2> QUESTION 8 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 9 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 10 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 11 </h2>
```
Comment (h8rry) [inline index.html:78]: Instead of hard coding these questions in HTML, you can dynamically generate them in JavaScript

File: js/application.js [hunk 3]
```diff
@@ -1,5 +1,42 @@
+
```
Comment (h8rry) [inline js/application.js:1]: Also still some indentation problems

**Comment by h8rry** [comment]
Looks like you are having a great progress!

File: css/custom.css (hunk 1 above)
Comment (Rachella) [inline css/custom.css:1]: corrected indentations

File: index.html (hunk 2 above)
Comment (Rachella) [inline index.html:78]: Okay, giving it a try with the new branch.

**Comment by h8rry** [comment]
Looks good. Merging.

**Pull request merged**